manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -53,9 +53,6 @@ Options SymLinksIfOwnerMatch
   # For OpenID-Connect /openid-connect is only served by mod_auth_openidc
   RewriteCond %%{REQUEST_URI} !^/openid-connect
 
-  RewriteRule ^/ http://ui:3000%%{REQUEST_URI} [P,QSA,L]
-  ProxyPassReverse / http://ui:3000/
-
   ProxyPass /api http://web-service:3000/api
```

Comment:
Does this API service receive any traffic after this change?  The rewrite conditions that were applied to `^/` are now applied to `/api` including `RewriteCond %%{REQUEST_URI} !^/api`.

Reply:
No actually the api service does not receive any traffic.

Reply:
Pushed a new commit. Api is now web-service again.